Yoast/wordpress-seo
# P3-594 Revert all Global Social Template changes from the admin

## Context


* Revert the Frontpage changes and put it back under SEO > Social > Facebook
* Revert the social appearance notifications with Open Graph disabled under SEO > Search Appearance > Content Types / Taxonomies / Archives.

## Summary


This PR can be summarized in the following changelog entry:

* Removes all the Global Social Templates-related changes from the admin

## Relevant technical choices:

* I've also reverted changes to the `Indexable_Home_Page_Builder` and `Indexable_Home_Page_Watcher` to make sure the changes from the form are updated correctly in the indexable for testing purposes.

## Test instructions

### Test instructions for the acceptance test before the PR gets merged
This PR can be acceptance tested by following these steps:

* Checkout this branch and build as usual (`$ composer install && yarn && grunt build`)
* Visit Settings > Reading and make sure "Your homepage displays" is set to "Your latest posts"
* Visit SEO > Social > Facebook and make sure Open Graph is enabled
* Visit SEO > Search Appearance > Content Types and note there is no Frontpage section anymore
* Visit SEO > Social > Facebook and note the Frontpage form is shown
* Fill in the Image, Title and Description for the Frontpage with something recognizable
* Go to the frontpage of your website and check the `og:image`, `og:title` and `og:description` are set with those values
* Visit SEO > Social > Facebook and disable the Open Graph meta data toggle
* Install the latest Premium plugin (trunk or RC)
* Visit SEO > Search Appearance > Content Types and note there is no notificiation shown about the frontpage or anything else related to social appearance
* Check the same on the other tabs, specifically Taxonomies and Archives.


### Test instructions for QA when the code is in the RC




QA can test this PR by following these steps:

**Prerequisites**

* Visit Settings > Reading and make sure "Your homepage displays" is set to "Your latest posts"
* Visit SEO > Social > Facebook and make sure Open Graph is enabled

**The Frontpage form**

* Visit SEO > Search Appearance > Content Types and note there is no Frontpage section anymore
* Visit SEO > Social > Facebook and note the Frontpage form is shown
* Fill in the Image, Title and Description for the Frontpage with something recognizable
* Go to the frontpage of your website and check the `og:image`, `og:title` and `og:description` are set with those values

**The social appearance notifications**

* Visit SEO > Social > Facebook and disable the Open Graph meta data toggle
* Install the latest Premium plugin (trunk or RC)
* Visit SEO > Search Appearance > Content Types and note there is no notificiation shown about the frontpage or anything else related to social appearance
* Check the same on the other tabs, specifically Taxonomies and Archives.

## Impact check

This PR affects the following parts of the plugin, which may require extra testing:

*

## UI changes


## Documentation


## Quality assurance


Fixes #
